src/editions/GatedEditionMinter.sol

```diff
@@ -1,43 +1,37 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.7;
 
-import { IEditionSingleMintable } from "@zoralabs/nft-editions-contracts/contracts/IEditionSingleMintable.sol";
+import { IEdition } from "nft-editions/interfaces/IEdition.sol";
+
 import { IShowtimeVerifier, SignedAttestation } from "src/interfaces/IShowtimeVerifier.sol";
 import { IGatedEditionMinter } from "./interfaces/IGatedEditionMinter.sol";
 import { TimeCop } from "./TimeCop.sol";
 
 contract GatedEditionMinter is IGatedEditionMinter {
     error NullAddress();
-    error TimeLimitReached(IEditionSingleMintable collection);
     error VerificationFailed();
 
     IShowtimeVerifier public immutable override showtimeVerifier;
-    TimeCop public immutable timeCop;
 
-    constructor(IShowtimeVerifier _showtimeVerifier, TimeCop _timeCop) {
-        if (address(_showtimeVerifier) == address(0) || address(_timeCop) == address(0)) {
+    constructor(IShowtimeVerifier _showtimeVerifier) {
+        if (address(_showtimeVerifier) == address(0)) {
             revert NullAddress();
         }
 
         showtimeVerifier = _showtimeVerifier;
-        timeCop = _timeCop;
     }
 
     /// @param signedAttestation the attestation to verify along with a corresponding signature
     /// @dev the edition to mint will be determined by the attestation's context
     /// @dev the recipient of the minted edition will be determined by the attestation's beneficiary
     function mintEdition(SignedAttestation calldata signedAttestation) public override {
-        IEditionSingleMintable collection = IEditionSingleMintable(signedAttestation.attestation.context);
-
-        if (timeCop.timeLimitReached(address(collection))) {
-            revert TimeLimitReached(collection);
-        }
+        IEdition collection = IEdition(signedAttestation.attestation.context);
 
```

Comment:
Just to verify, we can remove TimeCop from here because we moved it to the [edition](https://github.com/showtime-xyz/showtime-contracts/blob/main/src/editions/GatedEditionMinter.sol#L32-L34), correct?

Reply:
that's right! It's now supported natively in the Edition contract, so no need for that external registry 👌